src/containers/Projects/ProjectRulesAndGuide/Rules/Rules.scss

```diff
@@ -0,0 +1,63 @@
+.Rules {
+  &__heading {
+    color: var(--color-gray-900);
+    font-size: 36px;
```

Comment:
btw, we no longer hard code the `font-size` now, we `@extend` the font css from the pre-defined fonts located at the `_font-mixins.scss` file

<img width="555" alt="Screen Shot 2021-06-19 at 4 37 53 PM" src="https://user-images.githubusercontent.com/32864116/122636600-b459b700-d11c-11eb-8306-87309da5ca4c.png">

Usage:
<img width="409" alt="Screen Shot 2021-06-19 at 4 39 16 PM" src="https://user-images.githubusercontent.com/32864116/122636644-e539ec00-d11c-11eb-8528-a7e8f601c5be.png">

Reply:
Ah, I'll update it!